tt_eager/tt_dnn/op_library/eltwise_unary/eltwise_unary_op.hpp

```diff
@@ -122,6 +122,42 @@ struct UnaryWithParam {
     const auto attribute_values() const { return std::make_tuple(std::cref(this->op_type), std::cref(this->param)); }
 };
 
+inline UnaryWithParam string_to_unary_with_param(const std::string& name) {
+    if (name == "relu")
+        return UnaryWithParam{.op_type = UnaryOpType::RELU};
+    else if (name == "gelu")
+        return UnaryWithParam{.op_type = UnaryOpType::GELU, .param = static_cast<float>(true)};
```

Comment:
Why static_cast<float>(true) here and not 1.0f? Is true/false used as enum flag here?

Reply:
@ayerofieiev-tt Different unary ops have different reasons for having `param`. For example, `GELU` uses it as a boolean value that specifies whether it's fast and approximate or not. And on the other hand, `POW` uses `param` as the power.

But the `param` is always stored a float. I did a cast from bool here to communicate that it's not any float value.

We should potentially change it to store the right type in every case so that the code is more clear

Reply:
Thank you, got it. 

> We should potentially change it to store the right type in every case so that the code is more clear

+1, and switch to enums instead of bools where needed. Not in this PR